packages/components/src/components/PlaygroundSettings.tsx

```diff
@@ -43,7 +43,7 @@ function scaleTypeToSqScale(
     case "log":
       return SqLogScale.create(args);
     case "symlog":
-      return SqSymlogScale.create(args);
+      return SqSymlogScale.create({ constant: 10, ...args });
```

Comment:
My current understanding is that `constant: 1` is the most natural default and we should keep it.

Also, you used 10 here, but 0.1 in `fr/scale.ts`.

Reply:
Good point, that's a mistake